### PR TITLE
Grammatical updates, reformatting

### PR DESCRIPTION
**Pull request by har00ga**
Overhaul of several README components

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -94,13 +106,9 @@ if err != nil {
...
 
-### Supported data types
-- bool
-- uint8
-- uint16
-- uint32
-- uint64
-- slice
-- array
-- struct
```
Comment (prestonvanloon) [inline README.md]: I think we need these.

Comment (har00ga) [reply README.md]: I relocated this information to directly under the 'Usage examples' header:

**Notice:** SSZ supports `bool`, `uint8`, `uint16`, `uint32`, `uint64`, `slice`, `array`, `struct` and `pointer`data types.

If you'd like me to present it a different way or perhaps specify a little clearer, happy to modify it.

File: README.md [hunk 2]
```diff
@@ -1,36 +1,45 @@
...
+`HashTreeRoot`example:
 ```go
-// HashTreeRoot SSZ marshals a value and packs its serialized bytes into leaves of a Merkle trie -
-// then, it determines the Merkle root of the trie.
 func HashTreeRoot(val interface{}) ([32]byte, error)
 ````
 
-## Usage
+## Usage examples
+**Notice:** SSZ supports `bool`, `uint8`, `uint16`, `uint32`, `uint64`, `slice`, `array`, `struct` and `pointer`data types.
```
Comment (prestonvanloon) [inline README.md]:
```suggestion
**Notice:** SSZ supports `bool`, `uint8`, `uint16`, `uint32`, `uint64`, `slice`, `array`, `struct` and `pointer` data types.
```

File: README.md [hunk 3]
```diff
@@ -1,36 +1,45 @@
...
+[![Discord](https://user-images.githubusercontent.com/7288322/34471967-1df7808a-efbb-11e7-9088-ed0b04151291.png)](https://discord.gg/KSA7rPr)
+[![Gitter](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/prysmaticlabs/geth-sharding?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge)
+
+# Simple Serialize (SSZ)
+Simple Serialize is the serialization algorithm standard for all data structures common across Ethereum 2.0 client implementations. It is outlined in the official [Ethereum 2.0 specification](https://github.com/ethereum/eth2.0-specs/blob/master/specs/simple-serialize.md). 
+
+### Need assistance?
+A more indepth breakdown of SSZ is available in [this section](https://prysmaticlabs.gitbook.io/prysm/how-prysm-works/ethereum-2.0-serialization-simple-serialize-ssz) of our official documentation. If you still have questions, feel free to stop by either our [Discord](https://discord.gg/KSA7rPr) or [Gitter](https://gitter.im/prysmaticlabs/geth-sharding?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge) and a member of the team or our community will be happy to assist you.
```
Comment (rauljordan) [inline README.md]:
```suggestion
A more in-depth breakdown of SSZ is available in [this section](https://prysmaticlabs.gitbook.io/prysm/how-prysm-works/ethereum-2.0-serialization-simple-serialize-ssz) of our official documentation. If you still have questions, feel free to stop by either our [Discord](https://discord.gg/KSA7rPr) or [Gitter](https://gitter.im/prysmaticlabs/geth-sharding?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge) and a member of the team or our community will be happy to assist you.
```

**Comment by rauljordan** [comment]
Hi @har00ga thanks for the work! Just commit the two outstanding suggestions and then looks good to merge :)

**Comment by rauljordan** [comment]
Hi @har00ga could you kindly take a look at the small suggestions and commit them so we can merge this in? Thanks for the help!

**Comment by har00ga** [comment]
Hey there! Sorry about that, I don't seem to be receiving notifications properly these days, may have to delve into my settings. Committed both changes, thank you both for the review (and the patience). :)

**Pull request merged**